### DISCUSSION TITLE
RenderAsPdf

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5769] [Serenity Ideas]
![image](https://user-images.githubusercontent.com/24252615/120712722-ade70080-c4c9-11eb-93ff-f6c6ae183c2b.png)
![image](https://user-images.githubusercontent.com/24252615/120715143-db817900-c4cc-11eb-8c31-679bd3b66bf5.png)

I am getting an error like this in the RenderAsPdf function where could I be making a mistake

**Comment by eissa1005** [comment]
@havvanur160401046 
UtilityExePath is null
Because the programs to convert from html to pdf do not exist  => wkhtmltopdf.exe

**Reply by eissa1005** [reply]
![Untitled](https://user-images.githubusercontent.com/20481378/121387111-b9637d00-c94a-11eb-9344-2fbacb0aff62.png)

**Reply by mrajalko** [reply]
please see my discussion #5654
